src/components/Input/Input.js

```diff
@@ -167,6 +167,13 @@ const InputContainer = styled('div')`
   ${containerInlineStyles};
 `;
 
+export const inputStyles = [
```

Comment:
Ah, too bad we have to do this. ¯\_(ツ)_/¯

Reply:
My bad. This was leftover from my initial try to make the calendar input field look like `circuit-ui`'s input field, but it is the wrong approach and it wasn't needed, since the design is actually different :) 

I've removed it.